httpserver/Makefile

```diff
@@ -4,6 +4,7 @@ generatedfiles := \
 	bootstrap_dot_min_dot_css_.go \
 	bootstrap_dot_min_dot_js_.go \
 	jsontree_dot_js_.go \
+	jquery_dot_min_dot_js_.go \
```

Comment:
Not quite sure why you didn't preserve the order followed in .gitignore (or visa versa), but whatever... As you probably noticed, the entries are alphabetic so I guess this one is the correct ordering :-).

Reply:
Oh, sorry, I didn't notice any specific order...